src/node_buffer.cc

```diff
@@ -701,6 +928,50 @@ void IndexOfNumber(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void IsNull(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!HasInstance(args[0]))
+    return env->ThrowTypeError("first arg should be a Buffer");
+
+  ARGS_THIS(args[0].As<Object>())
+
+  uint32_t offset = args[1]->Uint32Value();
+  CHECK_LE(offset, obj_length);
+
+  char *ptr = obj_data + offset;
+
+  args.GetReturnValue().Set( ptr == NULL );
```

Comment:
This would always be false when offset > 0 if it weren't undefined behavior. :-/